src/main/kotlin/org/rust/cargo/runconfig/filters/RsBacktraceFilter.kt

```diff
@@ -27,16 +28,42 @@ import java.util.regex.Pattern
  */
 class RsBacktraceFilter(
     project: Project,
-    cargoProjectDir: VirtualFile,
+    cargoProjectDir: VirtualFile?,
     workspace: CargoWorkspace?
 ) : Filter {
+    private val backtraceItemFilters: List<RsBacktraceItemFilter> = run {
```

Comment:
Unfortunately, we cannot do such things because IDE store object instantiated for exception point so this filter always "remembers" state that project had at first `Analyze Stack Trace` action invocation.
Also, it leads to memory leak because we store link to `Workspace` and `VirtualFile` objects

Reply:
Fixed.